src/lib.rs

```diff
@@ -133,6 +133,7 @@ pub mod test;
 
 #[cfg(test)]
 mod tests {
+    #[cfg(feature = "rsa_signing")]
```

Comment:
In BoringSSL, all of the use of `BN_rand_range` and most of the uses of `BN_range` have been removed from bn_test.cc. I will merge those changes soon, before I merge this PR. After that, and after #247, you should be able to update this PR to completely remove ``BN_rand_range`. We'll still need `BN_rand` for bn_test.cc, but we can do as you did in your first commit: just move it to bn_test_lib.c. Similarly, if there are other `BN_*` functions that would then only be used in tests, we can move them in a similar way; we just need to make sure we sort everything into files according to the license.